packages/mdc-tab/_mixins.scss

```diff
@@ -36,6 +37,17 @@
   }
 }
 
+@mixin mdc-tab-ripple-color($color) {
```

Comment:
Should we call this mdc-tab-states-color given that it's invoking mdc-states?

Reply:
That's a good idea. 

Just an aside: the mixed naming between "ripple" and "states" keeps tripping me up.

Reply:
SGTM, done.